docs/guide/url_management.md

```diff
@@ -94,7 +97,7 @@ namespace eZ\Bundle\EzPublishCoreBundle\URLChecker;
 interface URLHandlerInterface
 {
     /**
-     * Validates given list of URLs.
+     * Validates a given list of URLs.
```

Comment:
```suggestion
     * Validates given list of URLs.
```
This is part of product code

Reply:
Is it how the inline comments look in the product or is it an example with comments added for clarity?

Reply:
This is an exact quote from code, see https://github.com/ezsystems/ezplatform-kernel/blob/master/eZ/Bundle/EzPublishCoreBundle/URLChecker/URLHandlerInterface.php#L12